### PR TITLE
Prettier eslint

### PR DESCRIPTION
**Pull request by lutangar**
Most changes here are due to prettier formatting and fixable eslint rules.

Both prettier and eslint now run :
- on pre commit (locally)
- during CI on a PR

> Merging a PR failing to pass the "test" check on Github is now forbidden !

/!\ All the following rules have been turned off but should be reactivated ASAP
```
      "no-unused-vars": "off",
      "unused-imports/no-unused-vars": "off",
      "array-callback-return": "off",
      "react-hooks/exhaustive-deps": "off",
      "jsx-a11y/alt-text": "off",
      "jsx-a11y/anchor-is-valid": "off",
      "@typescript-eslint/no-unused-vars": "off",
      "no-lone-blocks": "off",
      "eqeqeq": "off"
```

Especially these :
```
      "array-callback-return": "off",
      "react-hooks/exhaustive-deps": "off",
      "no-lone-blocks": "off",
      "eqeqeq": "off"
```

> Most of the rules were already present in the [package.json](https://github.com/datatlas-erasme/front/blob/prettier-eslint/package.json#L76-L116).

The `package.json` had to be moved to the root directory. This is a standard and some tools are quite lost when the `package.json` isn't there.

~I might need to adjust the docker builds tho...~ --> see https://github.com/datatlas-erasme/front/pull/79/commits/32f3391c2b68f502590afa29774b5e3d2fef401a commit body.

### CODE REVIEW

**Comment by Pipazoul** [comment]
On windows, many Delete `␍` issues displayed

`git config --global core.autocrlf false` did the job, but the repo had to be pulled again

**Comment by lutangar** [comment]
Hey @Pipazoul thanks for the slick merge ! However we should probably agree on some kind of process with the team.

Few possible suggestions :
- use Github review system : approve / request changes
- wait for X approvals before the PR may be merged : it may give some more time for others team members to express themself
- ... ?

Let's have a chat about this next time :)

File: src/components/buttons/interactiv-button/index.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 import { IconAdd } from '../../../assets/svg/IconAdd';
+import { ConfContext } from '../../../providers/ConfProvider';
 import { AddPoint, WarpperAddPoint } from './style';
 
-export default function AddButton({ bottomRightButtons }) {
-  const config = bottomRightButtons;
+export default function AddButton() {
+  const config = useContext(ConfContext).bottomRightButtons;
+
+  console.log('config', config);
```
Comment (lutangar) [inline src/components/buttons/interactiv-button/index.js:9]: /!\ Leftover debug

Comment (Pipazoul) [reply src/components/buttons/interactiv-button/index.js:9]: Removed

**Pull request merged**